mne/io/cnt/cnt.py

```diff
@@ -1,26 +1,106 @@
+    event table is separated from the main .cnt is not supported.
+
+    Parameters
+    ----------
+    fname: str
+        path to cnt file containing the annotations.
+
+    Returns
+    -------
+    onset : array of float, shape (n_annotations,)
+        The starting time of annotations in seconds after ``orig_time``.
+    duration : array of float, shape (n_annotations,)
+        Durations of the annotations in seconds.
+    description : array of str, shape (n_annotations,)
+        Array of strings containing description for each annotation. If a
+        string, all the annotations are given the same description. To reject
+        epochs, use description starting with keyword 'bad'. See example above.
+    """
+    # Offsets from SETUP structure in http://paulbourke.net/dataformats/eeg/
+    SETUP_NCHANNELS_OFFSET = 370
+    SETUP_RATE_OFFSET = 376
+    SETUP_EVENTTABLEPOS_OFFSET = 886
+
+    def _translating_function(offset, n_channels, event_type, n_bytes=2):
+        # n_bytes is related to _get_cnt_info's data_format parameter
+        # 'auto', 'int16', and 'int32'
+        if event_type == CNTEventType3:
+            offset *= n_bytes * n_channels
+        event_time = offset - 900 - (75 * n_channels)
+        event_time //= n_channels * n_bytes
+        return event_time - 1
+
+    with open(fname, 'rb') as fid:
+        fid.seek(SETUP_NCHANNELS_OFFSET)
+        (n_channels,) = unpack('<H', fid.read(calcsize('<H')))
+
+        fid.seek(SETUP_RATE_OFFSET)
+        (sfreq,) = unpack('<H', fid.read(calcsize('<H')))
+
+        fid.seek(SETUP_EVENTTABLEPOS_OFFSET)
+        (event_table_pos,) = unpack('<l', fid.read(calcsize('<l')))
+
+    with open(fname, 'rb') as fid:
+        teeg = _read_teeg(fid, teeg_offset=event_table_pos)
+
+    event_parser = _get_event_parser(event_type=teeg.event_type)
+
+    with open(fname, 'rb') as fid:
+        fid.seek(event_table_pos + 9)  # the real table stats at +9
+        buffer = fid.read(teeg.total_length)
+
+    my_events = list(event_parser(buffer))
+
+    if not my_events:
+        return Annotations(list(), list(), list(), None)
+    else:
+        onset = _translating_function(np.array([e.Offset for e in my_events],
+                                               dtype=float),
+                                      n_channels=n_channels,
+                                      event_type=type(my_events[0]))
+        duration = np.array([e.Latency for e in my_events], dtype=float)
+        description = np.array([str(e.StimType) for e in my_events])
+        return Annotations(onset=onset / sfreq,
+                           duration=duration,
+                           description=description,
+                           orig_time=None)
 
 
 @fill_doc
 def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
                  data_format='auto', date_format='mm/dd/yy', preload=False,
-                 verbose=None):
+                 stim_channel=True, verbose=None):
```

Comment:
Here it should be `None`

Reply:
Shouldn't default to True to not break people's code? At the beginning I just removed the whole stim. But talking to @agramfort it seems that we cannot get read of the stim in cnt until 0.20, unless we backport the addition of this `stim_channel=True` to 0.17.3 and then 0.18 to false/none and remove the stim channel.